networkx/linalg/spectrum.py

```diff
@@ -32,11 +32,23 @@ def laplacian_spectrum(G, weight="weight"):
     Notes
     -----
     For MultiGraph/MultiDiGraph, the edges weights are summed.
-    See to_numpy_array for other options.
+    See :func:`~networkx.convert_matrix.to_numpy_array` for other options.
 
     See Also
     --------
     laplacian_matrix
+
+    Examples
+    --------
+    The multiplicity of O as an eigenvalue of the laplacian matrix is equal
+    to the number of connected components of G.
+
+    >>> import numpy as np
+    >>> G = nx.graph_atlas(22) #This graph from the Graph Atlas has 3 connected components.
+    >>> eigvals = nx.laplacian_spectrum(G).round(10)
+    >>> print(len(eigvals) - np.count_nonzero(eigvals)) #Count zero eigvalues
+    3
```

Comment:
IMO the numpy operations actually make the example harder to understand. I'd vote to simply show the result:

```suggestion
    >>> nx.laplacian_spectrum(G)
    array([0., 0., 0., 2., 2.])
```